sourcecode/hub/tests/Browser/NdlaTest.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Browser;
```

Comment:
Incorrect namespace, should be `Tests\Browser`

Reply:
PhpStorm keeps doing this. Maybe we need a CI job that fails when classes aren't namespaced correctly.

Reply:
Maybe the namespaces aren't configured correctly in PhpStorm, you can check by going to Settings and Directories

Reply:
Also make sure the `For generated source` is checked